pages/tool_assembly/xnat_pic_tool_assembly.md

```diff
@@ -0,0 +1,52 @@
+---
+title: XNAT-PIC - Extending XNAT to Preclinical Imaging Centers
```

Comment:
Is it possible to reduce the length of this title? This makes more usable in the toolstable.